src/pages/Overview/OverviewCardLinks.tsx

```diff
@@ -15,21 +15,43 @@ class OverviewCardLinks extends React.Component<Props> {
   render() {
     return (
       <div style={{ marginTop: '10px' }}>
-        <Link to={`/graph/namespaces?namespaces=` + this.props.name} title="Graph">
-          <Icon type="pf" name="topology" style={{ paddingLeft: 10, paddingRight: 10 }} />
-        </Link>
-        <Link to={`/${Paths.APPLICATIONS}?namespaces=` + this.props.name} title="Applications list">
-          <Icon type="pf" name="applications" style={{ paddingLeft: 10, paddingRight: 10 }} />
-        </Link>
-        <Link to={`/${Paths.WORKLOADS}?namespaces=` + this.props.name} title="Workloads list">
-          <Icon type="pf" name="bundle" style={{ paddingLeft: 10, paddingRight: 10 }} />
-        </Link>
-        <Link to={`/${Paths.SERVICES}?namespaces=` + this.props.name} title="Services list">
-          <Icon type="pf" name="service" style={{ paddingLeft: 10, paddingRight: 10 }} />
-        </Link>
-        <Link to={`/${Paths.ISTIO}?namespaces=` + this.props.name} title="Istio Config list">
-          <Icon type="pf" name="template" style={{ paddingLeft: 10, paddingRight: 10 }} />
-        </Link>
+        <OverlayTrigger key="ot_graph" placement="top" overlay={<Tooltip id="tt_graph">Go to graph...</Tooltip>}>
+          <Link to={`/graph/namespaces?namespaces=` + this.props.name}>
+            <Icon type="pf" name="topology" style={{ paddingLeft: 10, paddingRight: 10 }} />
+          </Link>
+        </OverlayTrigger>
+        <OverlayTrigger key="ot_apps" placement="top" overlay={<Tooltip id="tt_apps">Go to applications...</Tooltip>}>
+          <Link to={`/${Paths.APPLICATIONS}?namespaces=` + this.props.name}>
+            <Icon type="pf" name="applications" style={{ paddingLeft: 10, paddingRight: 10 }} />
+          </Link>
+        </OverlayTrigger>
+        <OverlayTrigger
+          key="ot_workloads"
+          placement="top"
+          overlay={<Tooltip id="tt_workloads">Go to workloads...</Tooltip>}
+        >
+          <Link to={`/${Paths.WORKLOADS}?namespaces=` + this.props.name}>
+            <Icon type="pf" name="bundle" style={{ paddingLeft: 10, paddingRight: 10 }} />
```

Comment:
what is name=bundle? what does it refer to?

Reply:
That is the name for the icon, pf-bundle.  No change here.